src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -219,7 +219,8 @@ std::string KeyboardShortcuts::GetShortcutString(int32_t shortcut) const
 ScreenCoordsXY KeyboardShortcuts::GetKeyboardMapScroll(const uint8_t* keysState) const
 {
     ScreenCoordsXY screenCoords;
-    for (int32_t shortcutId = SHORTCUT_SCROLL_MAP_UP; shortcutId <= SHORTCUT_SCROLL_MAP_RIGHT; shortcutId++)
+    for (int32_t shortcutId = static_cast<int32_t>(KeyboardShortcut::ScrollMapUp);
+         shortcutId <= static_cast<int32_t>(KeyboardShortcut::ScrollMapRight); shortcutId++)
```

Comment:
Isn't it better to save it as the proper enum type and cast below? The `for` condition will become shorter.

Reply:
You cannot increment an enum type.